unlock-app/next.config.js

```diff
@@ -23,6 +23,9 @@ let requiredConfigVariables = {
 if (process.env.UNLOCK_URL)
   requiredConfigVariables.unlockUrl = process.env.UNLOCK_URL
 
+if (process.env.UNLOCK_STATIC_URL)
+  requiredConfigVariables.unlockStaticUrl = process.env.UNLOCK_STATIC_URL
```

Comment:
I am not sure this is the right approach because this bypasses the mechanism below which would fail deploys if this is unset. I think `UNLOCK_STATIC_URL` and `UNLOCK_URL` should be set above inside the `requiredConfigVariables` and left to `null`/`undefined` if unset. This means that they will be overriden by config in dev but if none is set we cannot deploy to production (which is good because that would break links!)